src/ng2-datepicker/locales/locale-manager.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from "@angular/core";
+import { EnglishLocale } from "./en-locale";
+import { RussianLocale } from "./ru-locale";
+
+export interface ILocale{
+    todayText(): string;
+    selectYearText(): string;
+    clearText(): string;
+    daysOfWeek(): string[];
+    langIdentifier(): string;
+    firstWeekdaySunday(): boolean;
+}
+
+export class ILocaleManager{
+    Resolve(locale: string): ILocale{
+        throw new Error("ILocaleManager is an pseudo interface. It cannot be used like a commin class.");
+    }
+}
+
+@Injectable()
+export class LocaleManager implements ILocaleManager{
+    private _locales: ILocale[] = [];
+    private _defaultLocale: ILocale = new EnglishLocale();
+    
+    public constructor(){
+        this._locales["en"] = this._defaultLocale;
+        this._locales["ru"] = new RussianLocale();
```

Comment:
i really don't like this, this should be done that we can use `sl` locale for example... or any other locale that is supported by `moment`. can you do a refactor of those please?